### ISSUE TITLE
Logging and Replay

### ISSUE THREAD

**Issue by MagentaJohn** [Quafadas/indigoLite#8]
Capture the moves made in a game, and perhaps also the ability to replay. This might be useful in trying to reproduce bugs.

**Comment by Quafadas** [comment]
I think the best way to do this, woudl be to caputre every single move as an increment to the game state.

**Comment by MagentaJohn** [comment]
Yes I agree. Capturing is easy and can be written to cache. However, what I really need is a way to store current game and load previous game for playback mode. I think we need a new "issue" which basically says we are going to implement the other half of the Tyrian Bridge (ie indigo-> html). I would then have an indigo button that takes the player back to the html environment where I can use the rich GUI available. I also need this bridge (indigo-> html) so I can start a new game with new names and parameters. (At the moment "new game " is implemented as a game reset with the same names and parameters). Once again, I would need your help because I don't understand the Tyrian Bridge stuff.
